release-notes/Cloud/DCP_change_log.md

```diff
@@ -10,6 +10,17 @@ The dataminer.services platform gets updated continuously. This change log can h
 
 A concurrency issue was introduced in CloudGateway 2.10.8 that could cause the CloudGateway to fail to respond to certain requests, such as creating a share. This issue has now been resolved.
 
+#### 17 May 2023 - Enhancement - CloudFeed 1.1.0 - Possibility for other processes to query the CloudFeed state [ID_35914]
+
+Other DataMiner processes can now check the state of DataMiner CloudFeed. For now, DataMiner Cube will use this to check if the Relation Learning feature is available.
+
+#### 17 May 2023 - Enhancement - CloudFeed 1.1.0 - CloudGateway as proxy for cloud connection [ID_35663]
+
+The CloudFeed process can now use CloudGateway as a proxy for the cloud connection. This way, CloudFeed can be used on systems that are not allowed to have outbound connections to the cloud.
+
+> [!NOTE]
```

Comment:
Same is valid for ArtifactDeployer, see https://github.com/SkylineCommunications/dataminer-docs/blob/6fe90cc2c7e2ce0f0417d16425772acb1021c2dc/release-notes/Cloud/DCP_change_log.md#26-april-2023---enhancement---artifactdeployer-145---artifactdeployer-no-longer-requires-direct-internet-access-id_36284